src/options/options.html

```diff
@@ -59,110 +59,156 @@
-							If a QR code is damaged or some parts are missing, it can often still be scanned.
-							The higher the value the more the machine is able to understand.
+					</div>
+				</div>
+				
+				<div class="item">
+					<div class="line">
+						<label data-i18n="__MSG_optionQrCodeBackgroundColor__" for="qrBackgroundColor">QR code background color: </label>
+						<input class="setting save-on-change trigger-on-update" type="color" id="qrBackgroundColor" name="qrBackgroundColor">
+					</div>
+				</div>
+				
+				<div class="item">
+					<div class="line">
+						<label data-i18n="__MSG_optionErrorCorrection__" for="qrErrorCorrection">Error correction level: </label>
+						<select id="qrErrorCorrection" class="setting save-on-change" name="qrErrorCorrection" size="0">
+							<option data-i18n="__MSG_optionEcLow__" value="L">Low (7%)</option>
+							<option data-i18n="__MSG_optionEcMedium__" value="M">Medium (15%)</option>
+							<option data-i18n="__MSG_optionEcQuartile__" value="Q">Quartile (25%)</option>
+							<option data-i18n="__MSG_optionEcHigh__" value="H">High (30%)</option>
+						</select>
+					</div>
+					<div class="line indent">
+						<span class="helper-text">
+							<span data-i18n="__MSG_optionErrorCorrectionDescr__">
+								If a QR code is damaged or some parts are missing, it can often still be scanned.
+								The higher the value the more the machine is able to understand.
+							</span>
+							<a data-i18n="__MSG_optionLearnMore__" data-i18n-href="__MSG_optionErrorCorrectionDescrLink__" href="https://en.wikipedia.org/wiki/QR_code#Error_correction">Learn more</a>
 						</span>
-						<a data-i18n="__MSG_optionLearnMore__" data-i18n-href="__MSG_optionErrorCorrectionDescrLink__" href="https://en.wikipedia.org/wiki/QR_code#Error_correction">Learn more</a>
-					</span>
-				</li>
-				<li>
-					<input class="setting save-on-change" type="checkbox" id="autoGetSelectedText" name="autoGetSelectedText">
-					<label data-i18n="__MSG_optionAutoGetSelectedText__" for="autoGetSelectedText">Automatically use text selected on website</label>
-				</li>
-				<li>
-					<input class="setting save-on-change" type="checkbox" id="monospaceFont" name="monospaceFont">
-					<label data-i18n="__MSG_optionUseMonospaceFont__" for="monospaceFont">Use monospace font</label>
-				</li>
-				<li>
-					<input class="setting save-on-change" type="checkbox" id="debugMode" name="debugMode">
-					<label data-i18n="__MSG_optionDebugMode__" for="debugMode">Enable debug mode</label>
-					<span data-i18n="__MSG_optionDebugModeDescr__" class="helper-text">This is only useful to get more detailed logging output in the console for reporting bugs, etc.</span>
-				</li>
-				<li>
-					<button data-i18n="__MSG_optionsResetButton__" type="button" id="resetButton">Reset all settings to defaults</button>
-				</li>
-			</ul>
+					</div>
+			</section>
+			<h1>TODO: Addon behavior</h1>
```

Comment:
It's possibly better to not make these bigger headers h1, but h2.

Reply:
my rationale is to reduce the amount of "defaults removing behavior", ul and li have those listing properties. i could add that back in